openapi/openapi.yml

```diff
@@ -1370,15 +1327,19 @@ components:
       schema:
         type: integer
         default: 10
-    postgres_database_id:
-      description: Postgres database ID
+    postgres_database_identifier:
+      description: Postgres database ID or name
       in: path
-      name: postgres_database_id
+      name: postgres_database_identifier
       required: true
       schema:
-        type: string
-        example: pgn30gjk1d1e2jj34v9x0dq4rp
-        pattern: '^pg[0-9a-hj-km-np-tv-z]{24}$'
+        oneOf:
+          - type: string
+            example: pgn30gjk1d1e2jj34v9x0dq4rp
+            pattern: '^pg[0-9a-hj-km-np-tv-z]{24}$'
```

Comment:
For backwards compatibility, you have to allow an optional `_` at the beginning.

Additionally, we should probably accept `[0-9a-tv-z]`.  The ubid code will allow `i`, `l`, and `o`, and turn them into `1`, `1`, and `0`.  I'm unsure why `u` isn't converted to `v`, seems like an oversight to not handle all 26 characters.  Maybe @pykello knows?

Reply:
Thanks. I think we aren't worrying about backwards compatibility any more, but let me know if I'm wrong there. Otherwise, I've updated here: https://github.com/ubicloud/ubicloud/pull/2907/commits/b2e7dcf1004ad4a3609bfc8e9b8a79fde061d845